### ISSUE TITLE
Installation error: undefined exports

### ISSUE THREAD

**Issue by ethan-pawl** [sangwon-hyun/flowsmooth#1]
Upon installing flowtrend, I get the message 

```
Error: package or namespace load failed for ‘flowtrend’ in namespaceExport(ns, exports):
 undefined exports: plot_1d_add_model, plot_1d_with_membership
```

which prevents installation. I believe this is because the two functions are defined not in the package subdirectory, but instead in the .Rmd files in the root folder of the repo.

**Comment by sangwon-hyun** [comment]
Good catch! I just got so used to using `minimal_eval = TRUE` while building the package using`litr` that I didn't catch  this. I'll fix it very soon.
